front/pages/home/HubSpotForm.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from "react";
+
+declare global {
+  interface Window {
+    hbspt: any;
+  }
+}
+
+export function HubSpotForm() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    const formContainer = document.getElementById("hubspotForm");
```

Comment:
not quite clear, but hubspotForm element does not exist at this time. (loading is true, so your element is not rendered, line 59) 
Maybe move this where it's needed ? (line 49) Or use a useRef instead ?